app/scripts/modules/core/help/helpField.component.ts

```diff
@@ -117,12 +117,30 @@ const helpFieldComponent: IComponentOptions = {
         <span class="small glyphicon glyphicon-question-sign"></span>
       </a>
     </div>
-  `
-};
+  `;
+}
+
+export class HelpFieldWrapperComponent implements IComponentOptions {
```

Comment:
Ah... very interesting pattern to avoid having to change all the cases for the `@` bindings... I think I like it? I think I like it.